README.rst

```diff
@@ -37,7 +37,7 @@ authorization are editable. To do this, all data about the workflow item
 is persisted into DB. **Hence, they can be changed without touching the
 code and re-deploying your application.**
 
-There is ordering aprovments for a transition functionality in
+There are ordering improvements for a transition functionality in
```

Comment:
Yes you are right. The feature is "lining up approvals". Ordering is not proper usage.